src/content_api/content_api_extract.py

```diff
@@ -137,7 +74,7 @@ def chunked_extract(nodes_srs, chunks_list, destination_dir):
 
 def extract_link_types(collection_links, content_item, related_page_links, page_links):
     if content_item is not None:
-        links = get_text(content_item['details'])
+        links = extract_from_details(content_item['details'])
```

Comment:
`function_type="links"` here to extract links not details?

Reply:
Needs to be updated.